cavityDisplayGUI.py

```diff
@@ -85,24 +91,34 @@ def __init__(self, parent=None, args=None):
 
                     severityPV = PV(cavityObject.pvPrefix + SEVERITY_SUFFIX)
                     statusPV = PV(cavityObject.pvPrefix + STATUS_SUFFIX)
+                    descriptionPV = PV(cavityObject.pvPrefix + DESCRIPTION_SUFFIX)
 
-                    # This line is meant to initialize the cavityWidget colors and shapes when first launched
+                    # These lines are meant to initialize the cavityWidget color, shape, and descriptionPV values when first launched
```

Comment:
Can we also add a line saying that if we don't have this initialization, the fields are blank until the PV changes hahahaha

Reply:
Added the comment:) Luckily, that shouldn't come up again though now that we've got that initialization line:D